docs/docs/examples/bi.md

```diff
@@ -29,12 +29,19 @@ Build a platform that ingests data from multiple sources (Postgres and Shopify),
     }
   }
 }%%
-    graph LR
-        Postgres ==> Airbyte
-        Shopify ==> Airbyte
-        Airbyte ==> BigQuery
-        BigQuery ==> Looker
-        BigQuery ==> Hex
+graph LR
+    AB[<img src='/images/examples/icons/airbyte.svg' width='50' height='50' /> Airbyte]
```

Comment:
@dehume, using the integration icons appears to be working locally for me.

```
    AB[<img src='/images/integrations/airbyte.svg' width='50' height='50' /> Airbyte]
```

Reply:
Is there any reason we woudn't want the reference architecture updated when the integration icon was updated?